### DISCUSSION TITLE
How to make detail page without form

### DISCUSSION THREAD

**Discussion by buenojosias** [filamentphp/filament#5224] [General]
When we run the command "php artisan make:filament-resource Model --view", it generates a read-only form with the fields defined in the resource.
But I would like to create a page similar to the one in the image. Is there a way to do this? I looked for a tutorial on several sites and I couldn't find it, not even in the documentation.
![description-lists 05-two-column-in-card-xl](https://user-images.githubusercontent.com/66849846/206878127-4e3ae606-266a-4bf6-b855-38927e3d7794.png)

**Comment by ArtMin96** [comment]
[https://filamentphp.com/docs/2.x/admin/pages/getting-started](https://filamentphp.com/docs/2.x/admin/pages/getting-started)
